### ISSUE TITLE
Website Accessibility-WCAG 4.1.2: Nested interactive controls are not announced by screen readers (li[aria-controls="example-1-css"])

### ISSUE THREAD

**Issue by A11Ybot-OP** [sass/sass#3233]
#### Issue
`Nested interactive controls are not announced by screen readers` ([nested-interactive](https://dequeuniversity.com/rules/axe/4.3/nested-interactive?application=msftAI))
#### Target application
[Sass: Sass Basics](https://sass-lang.com/guide)
#### Element path
li[aria-controls="example-1-css"]
#### Snippet
```<li class="ui-tabs-tab css-tab ui-corner-top ui-state-default ui-tab" role="tab" tabindex="-1" aria-controls="example-1-css" aria-labelledby="ui-id-3" aria-selected="false" aria-expanded="false">```
#### How to fix
    Fix any of the following:
      Element has focusable descendants
#### Environment
Microsoft Edge version 96.0.1054.62

**Comment by A11Ybot-OP** [comment]
There are 15 additional places with `nested-interactive` accessibility issues.
